pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -147,7 +147,62 @@ func (b *AlicloudBotanist) GenerateKubeSchedulerConfig() (map[string]interface{}
 
 // GenerateEtcdBackupConfig returns the etcd backup configuration for the etcd Helm chart.
 func (b *AlicloudBotanist) GenerateEtcdBackupConfig() (map[string][]byte, map[string]interface{}, error) {
-	return map[string][]byte{}, map[string]interface{}{}, nil
+	bucketName := "bucketName"
+	storageEndPoint := "storageEndPoint"
+
+	tf, err := b.NewBackupInfrastructureTerraformer()
+	if err != nil {
+		return nil, nil, err
+	}
+	stateVariables, err := tf.GetStateOutputVariables(bucketName, storageEndPoint)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	secretData := map[string][]byte{
+		StorageEndpoint: []byte(stateVariables[storageEndPoint]),
+		AccessKeyID:     b.Seed.Secret.Data[AccessKeyID],
+		AccessKeySecret: b.Seed.Secret.Data[AccessKeySecret],
+	}
+
+	backupConfigData := map[string]interface{}{
+		"schedule":         b.Operation.ShootBackup.Schedule,
```

Comment:
```suggestion
		"schedule":         b.ShootBackup.Schedule,
```

Reply:
Addressed the comment. However, noticed in `controlplane.go` of other type cloud botanists, `Operation` is also called first and then `ShootBackup` is called from it.